README.md

```diff
@@ -163,19 +163,19 @@ For example, developers may use this feature to bind a Habushu module's `compile
 ```
 
 ### Leveraging the containerize-dependencies Goal to Prepare a Containerized Virtual Environment ###
-If the execution is specified (does not run by default), the `containerize-dependencies` goal will collect a single `habushu` dependency specified the project's `pom.xml`, including all transitive habushu-packaged dependencies. After collecting the set of necessary dependencies, Habushu will copy the project files to the build directory under `venv-support`, while preserving the original structure of the dependency modules to ensure that any path-based dependencies can be leveraged as-is. This directory can then be copied onto a Docker container and used to create a virtual environment capable of running the target Habushu project.
+If the execution is specified (does not run by default), the `containerize-dependencies` goal will collect a single `habushu` dependency specified the project's `pom.xml`, including all transitive habushu-packaged dependencies. After collecting the set of necessary dependencies, Habushu will copy the project files to the build directory under `containerize-support`, while preserving the original structure of the dependency modules to ensure that any path-based dependencies can be leveraged as-is. This directory can then be copied onto a Docker container and used to create a virtual environment capable of running the target Habushu project.
 
 ```xml
 	<plugin>
 		<groupId>org.technologybrewery.habushu</groupId>
 		<artifactId>habushu-maven-plugin</artifactId>
 		<configuration>
-		...
+          <dockerfile>${project.basedir}/src/main/docker/Dockerfile</dockerfile>
```

Comment:
Think this needs to be spaced in more, looks like the indentation is off

Reply:
It's because the surrounding lines use tabs, and this line uses spaces. Might just convert them all to spaces.

Reply:
yeah.. reformat the doc with spaces